### ISSUE TITLE
Fix lint error before releases

### ISSUE THREAD

**Issue by krt16s** [y20k/transistor#29]
A clean checkout of 1.1.6 gives me a lint error:

````
:app:lintVitalRelease/home/krt/src/fdroiddata/build/org.y20k.transistor/app/src/main/res/values-sr/strings.xml:4: Error: The resource string "app_name" has been marked as translatable="false" [ExtraTranslation]
    <string name="app_name">Транзистор</string>
            ~~~~~~~~~~~~~~~

   Explanation for issues of type "ExtraTranslation":
   If a string appears in a specific language translation file, but there is
   no corresponding string in the default locale, then this string is probably
   unused. (It's technically possible that your application is only intended
   to run in a specific locale, but it's still a good idea to provide a
   fallback.).

   Note that these strings can lead to crashes if the string is looked up on
   any locale not providing a translation, so it's important to clean them
   up.

1 errors, 0 warnings
````

while I can (and did) forcefully ignore them in gradle's `android` section with

````
lintOptions {
   abortOnError false
}
````
This is a hack. A release should build with out modification. Either fix those issues on your side or (just in case) apply the workaround in your build.gradle :)

**Comment by y20k** [comment]
Thanks for pointing that out. And sorry for the inconvenience that version 1.1.6 seems cause with F-Droid. 

Problem is/was: I tried to undo some changes I made for the next version of Transistor to make the older version 1.1.6 build. I overlooked that I now got a new translation that was in conflict with a translateable=false flag. This translation originally was not part of 1.1.6 and it was not tested for release yet.

In short: The lint error should be gone in the next version which will be released next week.

Best wishes (and thank you for "manually" building 1.1.6)

**Comment by krt16s** [comment]
Still is there on 1.2.0, fixed it up on our side.

**Comment by y20k** [comment]
Hi there. Please let me know if there are still any errors. On my setup (Android Studio) the app builds an APK without problems. Best wishes.

**Comment by krt16s** [comment]
Works fine, thanks.

**Comment by y20k** [comment]
I was asking because I was expecting that there was still anything wrong with this repo. I published v1.2.2 five days ago and it does not show up on F-Droid. 

I can download and compile the source on my System. There are Lint warnings (overdraw, left-right-stuff) but there are no Lint errors.

Sorry to bother.

**Comment by meonwax** [comment]
https://f-droid.org/repository/browse/?fdid=org.y20k.transistor

> Version 1.2.2 (Cygnet Committee) - Added on 2016-02-12

**Comment by y20k** [comment]
Uups. Should have checked that. THX
